src/components/feed/FeedTree/Node.tsx

```diff
@@ -171,12 +172,36 @@ const NodeMemoed = React.memo(Node)
 
 const NodeWrapper = (props: NodeWrapperProps) => {
   const { data, overlayScale } = props
+  const [outputSize, setOutputSize] = useState<number>()
+
+  const pluginInstances = useTypedSelector(
+    state => state.instance.pluginInstances.data
+  )
+
+  React.useEffect(() => {
+    const pluginInstance = pluginInstances?.find((instance) => {
+      return instance.collection.items[0].data[0].value === data.id
+    })
+
+    if (pluginInstance) {
+      const pluginFiles = getPluginFiles(pluginInstance).next().value
+      if (pluginFiles instanceof Promise<any[]>) {
+        pluginFiles.then((files) =>{
+          const fileSizes = files.map((file) => {
```

Comment:
does not account for pagination. I checked the helper you're calling and it hard-codes 200 files. We need to think of how to better handle pagination across this entire project, maybe using async iterable

https://github.com/FNNDSC/ChRIS_ui/blob/68b9037c6236828a50df7ac0ca45799dca90896a/src/store/resources/saga.ts#L23-L29